integration-cli/docker_utils.go

```diff
@@ -303,26 +292,26 @@ func getAllNetworks() ([]types.NetworkResource, error) {
 	return networks, nil
 }
 
-func deleteAllPlugins() error {
+func deleteAllPlugins(c *check.C) {
 	plugins, err := getAllPlugins()
-	if err != nil {
-		return err
-	}
-	var errors []string
+	c.Assert(err, checker.IsNil)
+	var errs []string
 	for _, p := range plugins {
-		status, b, err := sockRequest("DELETE", "/plugins/"+p.Name+":"+p.Tag+"?force=1", nil)
+		pluginName := p.Name
+		tag := p.Tag
+		if tag == "" {
+			tag = "latest"
+		}
+		status, b, err := sockRequest("DELETE", "/plugins/"+pluginName+":"+tag+"?force=1", nil)
 		if err != nil {
-			errors = append(errors, err.Error())
+			errs = append(errs, err.Error())
 			continue
 		}
-		if status != http.StatusNoContent {
-			errors = append(errors, fmt.Sprintf("error deleting plugin %s: %s", p.Name, string(b)))
+		if status != http.StatusOK {
```

Comment:
plugin remove doesn't answer `http.StatusNoContent` (maybe it should but it does not)